api/HTMLDocument.json

```diff
@@ -20,7 +20,7 @@
             "version_added": "4"
           },
           "ie": {
-            "version_added": "≤6"
+            "version_added": "4"
```

Comment:
This matches https://developer.mozilla.org/en-us/docs/Web/API/Window/document#browser_compatibility.